data/account/participationRegistry.go

```diff
@@ -82,6 +82,22 @@ type ParticipationRecord struct {
 	Voting *crypto.OneTimeSignatureSecrets
 }
 
+// StateProofKey is a placeholder for the real state proof key type.
+// PKI TODO: Replace this with a real object.
+type StateProofKey []byte
+
+// ParticipationRecordForRound adds in the per-round state proof key.
+type ParticipationRecordForRound struct {
```

Comment:
The struct name is abit awkward. How do you feel about RoundParticipationSigner ?

Reply:
because this struct contains more than just a signer, I wouldn't use the word signer in it.
do we plan to use this struct as a return value from the `keymanager` interface? or should we rather abstract the use of the registry and return a new one?

Reply:
The registry return values are deep copies of the data, so it's safe to use these objects globally. The current interfaces are sufficient for master, so any `keymanager` changes can be designed for state proofs.